torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -1865,9 +1863,7 @@ std::vector<Val*> Index::getGlobalConsumerStridedIndices(
   {
     int stride_i = 0;
     for (const auto i : c10::irange(root_dom.size())) {
-      if (root_dom[i]->isReduction() ||
-          root_dom[i]->getIterType() == IterType::BroadcastWithoutStride ||
-          root_dom[i]->isStride()) {
+      if (root_dom[i]->isReduction() || root_dom[i]->isStride()) {
```

Comment:
At line 1316, the stride of a broadcast domain is set as `oneVal`, which seems inconsistent compared to here.

Reply:
Yeah, code is inconsistent, but the only thing that actually matters is the `stride_i++` in either of these. Will try to make them consistent.

Reply:
Since I effectively removed `BoadcastWithoutStride` I think the change on 1316 is the inconsistent one.

Reply:
Why is there an `isStride` in this conditional but not the one in the global producer? Does it not follow the same pattern as `isReduction` in indexing?

Reply:
Stride IDs are removed in rfactor domains, so no stride IDs should appear in producer rfactor domains. Having the same conditional in the producer indexing cases should not break, but it's just not necessary.